net-print/cnijfilter/cnijfilter-3.80.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+PS="${PN}-source-${PV}-1"
```

Comment:
We usually name this `MY_P`, and I suggest you follow suit for clarity.

Reply:
Fixed